src/engraving/libmscore/textedit.cpp

```diff
@@ -589,7 +589,7 @@ bool TextBase::edit(EditData& ed)
         score()->undo(new InsertText(_cursor, s), &ed);
 
         int startPosition = cursor->currentPosition();
-        notifyAboutTextInserted(startPosition, startPosition + s.size(), s);
+        notifyAboutTextInserted(startPosition, startPosition + static_cast<int>(s.size()), s);
```

Comment:
Maybe the same can be done here with `notifyAboutTextInserted`.

Reply:
Not today anymore

Reply:
Fair enough, seems like that would involve larger changes...

Reply:
That's just one reason, the other is that I have a life too 😉